File: Practice/src/coding_basic_training/LastTwoElements.java
package coding_basic_training;

import java.util.Arrays;

public class LastTwoElements {
//	마지막 두 원소
	public int[] solution(int[] num_list) {
		int len = num_list.length;
		int[] answer = Arrays.copyOf(num_list, len + 1);
		int a = num_list[len - 2];
		int b = num_list[len - 1];
		if (a >= b) {
			answer[len] = b * 2;
		} else {
			answer[len] = b - a;
		}
		return answer;
	}
}

File: Practice/src/coding_basic_training/LastTwoElements.js
function solution(num_list) {
    let a = num_list[num_list.length - 2];
    let b = num_list[num_list.length - 1];
    num_list.push(a >= b ? b * 2 : b - a);
    return num_list;
    // const [a, b] = [...num_list].reverse();
    // return [...num_list, a > b ? a - b : a * 2];
}
